package/MDAnalysis/coordinates/dcdtimeseries.pyx

```diff
@@ -115,15 +115,21 @@ def __read_timecorrel(object self, object atoms, object atomcounts, object forma
     for i from 0 <= i < n_frames:
         if (step > 1 and i > 0):
             # Check if we have fixed atoms
-            # XXX not done
```

Comment:
This comment said "We should check for fixed atoms here but we have not implemented it".

I don't think that you are addressing fixed atoms anywhere (it's an obscure feature in CHARMM DCD files where only the first frame contains the positions of the frozen atoms and all later frames do not have these positions, which are unchanging.) Therefore, *please leave this comment in*.